controllers/book.js

```diff
@@ -66,21 +82,30 @@ exports.getBookByQuery = (req, res) => {
 exports.updateBookById = (req, res) => {
     const bookId = req.params.id;
     Book.findByIdAndUpdate(bookId, req.body)
-        .then(res.send(`Record with id:${bookId} --> updated successfully!`))
+        .then(book => {
+            res.status(200).json({
+                book // WHY? - response is not updated book
+            })
+        })
         .catch(
             err => {
-                console.log(err)
-                res.status(404).send(err.message)
+                res.status(404).json({
+                    message: 'Server error'
+                })
             });
 };
 
 
+// find book by id and delete
 exports.deleteBookById = (req, res) => {
-    const bookId = req.params.id;// find book by id and delete
-    Book.findOneAndDelete({ _id: bookId })
-        .then(res.send(`Record with id:${bookId} --> deleted from DB!`))
-        .catch(err => {
-            console.log(err);
-            res.status(500).send();
+    const bookId = req.params.id;
+    Book.findOneAndDelete({
```

Comment:
Also you can use method findByIdAndDelete

Reply:
modified from "FindOneAndDelete" into "findByIdAndDelete"